src/constants.js

```diff
@@ -225,60 +238,65 @@ export const styles = {
     fontSize: 12
   },
 
-  dataDisplayHeaderLabelSelected: {
-    backgroundColor: labelColor,
-    color: "yellow"
-  },
-  dataDisplayHeaderFeatureSelected: {
-    backgroundColor: featureColor,
-    color: "yellow"
-  },
-  dataDisplayHeaderSelected: {
-    color: "yellow",
-    backgroundColor: "black"
+  dataDisplayHeader: {
+    paddingLeft: 20,
+    textAlign: "right",
+    position: "sticky",
+    top: 0,
+    backgroundColor: "white",
+    color: "#4d575f",
+    borderStyle: "solid",
+    borderWidth: 1,
+    borderColor: "white", // #c6cacd",
```

Comment:
tiny nit: Do we need these little comments here and line 171? I'm not sure when we're using words, when hex and when rgb and why.

Reply:
Oh, I will remove this detritus.